tests/src/Template/Provider/BaseProviderTest.php

```diff
@@ -276,15 +292,25 @@ private static function createPackage(
         string $name,
         string $type = 'project-builder-template',
         string $prettyVersion = '1.0.0',
-    ): Package\Package {
+    ): Package\CompletePackage {
         $versionParser = new VersionParser();
 
-        $package = new Package\Package($name, $versionParser->normalize($prettyVersion), $prettyVersion);
+        $package = new Package\CompletePackage($name, $versionParser->normalize($prettyVersion), $prettyVersion);
```

Comment:
@eliashaeussler Please let me know if that's too invasive. I'd revert this change and refactor `createAbandonedPackage()` accordingly.

Reply:
Fine for me.